providers/summary/scancode.js

```diff
@@ -81,15 +90,31 @@ class ScanCodeSummarizer {
       .filter(e => e)
   }
 
-  _toExpression(licenses) {
-    if (!licenses) return null
-    const list = setToArray(licenses)
+  _joinExpressions(expressions) {
+    if (!expressions) return null
+    const list = setToArray(expressions)
     if (!list) return null
-    return list
-      .map(SPDX.normalize)
-      .filter(x => x)
-      .join(' AND ')
+    return list.join(' AND ')
   }
+
+  _createExpressionFromRule(rule, licenseKey) {
+    if (rule && rule.rule_relevance < 50) return null
```

Comment:
this is the cutoff for scancode score

Reply:
I really wonder how many license values this is going to eliminate. 50 feels a bit low. Sort of "Meeeh, maybe, maybe not?"

@pombredanne  what do you think a reasonable `rule_relevance` value is?

Reply:
thanks for the info. In this case, I will update to use `license.score` rather than `license.matched_rule.rule_relevance`